presto-main/src/main/java/com/facebook/presto/server/ServerMainModule.java

```diff
@@ -510,6 +511,12 @@ private static void bindFailureDetector(Binder binder, boolean coordinator)
                 {
                     return ImmutableSet.of();
                 }
+
+                @Override
+                public State getState(HostAddress hostAddress)
+                {
+                    return State.UNKNOWN;
```

Comment:
This branch should only be triggered when it's NOT on coordinator right? (Since `coordinator` is false). So my understanding is that this is on the worker, and there is no failure detector on worker so it looks to me `UNKNOWN` is the only answer :(